server/glsp-server-websocket/src/main/java/com/eclipsesource/glsp/server/websocket/GLSPConfigurator.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright (c) 2019 EclipseSource and others.
+ *  
+ *   This program and the accompanying materials are made available under the
+ *   terms of the Eclipse Public License v. 2.0 which is available at
+ *   http://www.eclipse.org/legal/epl-2.0.
+ *  
+ *   This Source Code may also be made available under the following Secondary
+ *   Licenses when the conditions for such availability set forth in the Eclipse
+ *   Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ *   with the GNU Classpath Exception which is available at
+ *   https://www.gnu.org/software/classpath/license.html.
+ *  
+ *   SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ******************************************************************************/
+package com.eclipsesource.glsp.server.websocket;
+
+import java.util.List;
+import java.util.ServiceLoader;
+
+import javax.websocket.Extension;
+import javax.websocket.HandshakeResponse;
+import javax.websocket.server.HandshakeRequest;
+import javax.websocket.server.ServerEndpointConfig;
+import javax.websocket.server.ServerEndpointConfig.Configurator;
+
+import org.eclipse.jetty.websocket.jsr356.server.ContainerDefaultConfigurator;
+
+import com.google.inject.Injector;
+
+public class GLSPConfigurator extends Configurator {
+	private Configurator containerConfigurator;
+	private Injector injector;
+
+	public GLSPConfigurator(Injector injector) {
+		this.injector = injector;
+	}
+
+	Configurator getContainerConfigurator() {
+		if (containerConfigurator == null) {
+			containerConfigurator = ServiceLoader.load(javax.websocket.server.ServerEndpointConfig.Configurator.class)
```

Comment:
Afais, you don't need to mention the entire qualified name of the class ServerEndpointConfig.Configurator.